src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+$days-of-week: (mon, tue, wed, thu, fri, sat, sun);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
```

Comment:
The `body` selector should have a newline after it, according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), specifically 'Each selector should be on a new line'.